src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/interactivemaps/InteractiveMapPageObject.java

```diff
@@ -503,4 +508,12 @@ public void verifyPoiPointDescription() {
     PageObjectLogging.log("verifyPoiPointDescription",
                           "Poi description section is displayed", true);
   }
+
+  public void verifyPontoGetRequest(NetworkTrafficInterceptor networkTab) {
+    if(networkTab.searchRequestUrlInHar("maps.wikia-services.com")) {
```

Comment:
Basically, the test wants to check from Network tab if there is a request to maps.wikia-services.com right? Is this enough to assure the request from Ponto came?